client-spark/common/src/main/java/org/apache/spark/shuffle/handle/ShuffleHandleInfoBase.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.shuffle.handle;
+
+import java.io.Serializable;
+
+import org.apache.uniffle.common.RemoteStorageInfo;
+
+public abstract class ShuffleHandleInfoBase implements ShuffleHandleInfo, Serializable {
```

Comment:
`ShuffleHandleInfoBase` ->  `BaseShuffleHandleInfo`?

Reply:
I hope the prefix of ShuffleHandleInfo could be placed in a near group,

Reply:
Base class is a more common name.

Reply:
I like `BaseShuffleHandleInfo`, but a class name like `{Interface}Base` is common practice for a base implementation of an interface `{Interface}`.

Reply:
Let's reserve this name.